contrib/ipp/CMakeLists.txt

```diff
@@ -46,7 +46,6 @@ endif()
 
 list( APPEND sources ${srcs})
 enable_language( C ASM )
-set( CMAKE_ASM_EXTENSIONS OFF )
```

Comment:
-DCMAKE_ASM_COMPILER=clang-8 flag delete CMake flags

Reply:
I still don't see this in commit message.

Reply:
Now it's done